src/sourmash/minhash.py

```diff
@@ -543,6 +543,18 @@ def contained_by(self, other, downsample=False):
 
         return self.count_common(other, downsample) / len(self)
 
+    def max_containment(self, other, downsample=False):
+        """\
+        Calculate maximum containment.
+        """
+        if not (self.scaled and other.scaled):
+            raise TypeError("can only calculate containment for scaled MinHashes")
+        min_denom = min((len(self), len(other)))
+        if not min_denom:
+            return 0.0
```

Comment:
this should really never happen, right? Or could an empty sketch or aggressive downsampling trigger this?

Reply:
exactly - empty sketch, perhaps triggered by aggressive downsampling.

In 18b4f780, I've added two tests, `test_search_containment_s10_sbt_empty` and `test_search_max_containment_s10_sbt_empty `, that trigger this condition. Thanks!

Reply:
thx!